db/core/ecmcMotorRecord.template

```diff
@@ -315,46 +309,45 @@ record(mbbi,"$(PREFIX)$(MOTOR_NAME)-HomProc-RB") {
     field(FFST, "ManSetPos")
 }
 
-record(ai,"$(PREFIX)$(MOTOR_NAME)-HomPos-RB") {
+record(ai, "$(PREFIX)$(MOTOR_NAME)-HomPos-RB") {
     field(DESC, "Home Position")
     field(DTYP, "asynFloat64")
     field(PREC, "$(PREC)")
     field(INP,  "@asyn($(MOTOR_PORT),$(AXIS_NO))HomPos-RB")
     field(SCAN, "I/O Intr")
 }
 
-record (asyn, "$(PREFIX)$(MOTOR_NAME)-$(MOTOR_PORT)-asyn")
-{
+record (asyn, "$(PREFIX)$(MOTOR_NAME)-$(MOTOR_PORT)-asyn") {
 }
 
 ################################################################
 # Sync and stop on interlock
-record(calcout,"$(PREFIX)$(MOTOR_NAME)-MR-SyncTrgH2L"){
-  #field(PINI, "1")
-  field(DESC, "Trigg MR Sync")
-  field(SCAN, "Passive")
-  field(INPA,  "$(PREFIX)$(MOTOR_NAME)-SumIlockBwd CP")
-  field(INPB,  "$(PREFIX)$(MOTOR_NAME)-SumIlockFwd CP")
-  field(INPC,  "1")
-  field(CALC,  "A && B")
-  field(OCAL,  "C")
-  field(DOPT,  "Use OCAL")
-  field(OOPT,  "Transition To Zero")
-  field(OUT,   "$(PREFIX)$(MOTOR_NAME)-MR-SyncTrgSeq PP")
-}
-
-record(calcout,"$(PREFIX)$(MOTOR_NAME)-MR-SyncTrgL2H"){
-  #field(PINI, "1")
-  field(DESC, "Trigg MR Sync")
-  field(SCAN, "Passive")
-  field(INPA,  "$(PREFIX)$(MOTOR_NAME)-SumIlockBwd CP")
-  field(INPB,  "$(PREFIX)$(MOTOR_NAME)-SumIlockFwd CP")
-  field(INPC,  "1")
-  field(CALC,  "A && B")
-  field(OCAL,  "C")
-  field(DOPT,  "Use OCAL")
-  field(OOPT,  "Transition To Non-zero")
-  field(OUT,   "$(PREFIX)$(MOTOR_NAME)-MR-SyncTrgSeq PP")
+record(calcout, "$(PREFIX)$(MOTOR_NAME)-MR-SyncTrgH2L") {
+    #field(PINI, "1")
```

Comment:
If you're fixing the formatting here you could consider removing this (same in the following record def) - your call 🔧